test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1707,6 +1707,146 @@ UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_asymmetric_ib, rcv, "rc_v")
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_asymmetric_ib, rcx, "rc_x")
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_wireup_asymmetric_ib, ib, "ib")
 
+class test_ucp_wireup_reconfigure : public test_ucp_wireup {
+protected:
+    void verify_transport(const std::string &tl_name)
+    {
+        const ucp_ep_config_t *config = ucp_ep_config(sender().ep());
+        const auto tl_count           = count_resources(sender(), tl_name);
+        unsigned found_count          = 0;
+        size_t rma_bw_count           = 0;
+
+        for (int i = 0; i < config->key.num_lanes; ++i) {
+            const auto lane = config->key.rma_bw_lanes[i];
+
+            if (lane == UCP_NULL_LANE) {
+                break;
+            }
+
+            rma_bw_count++;
+            const char *transport =
+                    ucp_ep_get_tl_rsc(sender().ep(), lane)->tl_name;
+            if (tl_name == transport) {
+                found_count++;
+            }
+        }
+
+        EXPECT_GE(found_count, std::min(tl_count, rma_bw_count));
+    }
+
+    void send_before(const std::string &tl_name)
```

Comment:
Maybe rename these funcs to represent what they do (and not which phase of the test they are).

i.e. use terms like connect&send / wireup&send, etc.